application_test.go

```diff
@@ -297,6 +297,20 @@ func TestApplications(t *testing.T) {
 	assert.Equal(t, len(applications.Apps), 1)
 }
 
+func TestApplicationsEmbedTaskStats(t *testing.T) {
+	endpoint := newFakeMarathonEndpoint(t, nil)
+	defer endpoint.Close()
+
+	v := url.Values{}
+	v.Set("embed", "apps.taskStats")
+	applications, err := endpoint.Client.Applications(v)
+	assert.NoError(t, err)
+	assert.NotNil(t, applications)
+	assert.Equal(t, len(applications.Apps), 1)
+	assert.NotNil(t, applications.Apps[0].TaskStats)
+	assert.Equal(t, applications.Apps[0].TaskStats["startedAfterLastScaling"].Stats.Counts["healthy"], 1)
+}
```

Comment:
Seems like this comment got lost somehow:

Can we add an assertion for TaskStats.LifeTime too please?